src/App.js

```diff
@@ -1,27 +1,140 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
```

Comment:
- [ ] When I try to run the project according to instructions, webpack fails to compile:

![image](https://github.com/bhushan354/Full-Stack-Capstone-Frontend/assets/57421823/9547b55c-6684-46f8-805d-948977eb5538)

![image](https://github.com/bhushan354/Full-Stack-Capstone-Frontend/assets/57421823/c9b27fea-cdf0-40cd-ab27-493610787575)

Kindly, fix the dependencies so the app runs as intended. 👍 

_note: this means I wasn't able to access the project's main functionality.  This will be taken a look at the next review._